modules/mobile_connection.lua

```diff
@@ -44,7 +46,7 @@ function module.mt.__index:OnInputData(func)
     local msg = protocol_handler:Parse(binary)
     for _, v in ipairs(msg) do
       -- After refactoring should be moved in mobile session
-      xmlReporter:LOG("SDLtoMOB", v)
+      atf_logger.LOG("SDLtoMOB", v)
```

Comment:
@okozlovlux You replaced xml reporter on atf logger. Are you sure that this information should not also be kept in xml file?